modules/colombiaBidAdapter.js

```diff
@@ -0,0 +1,111 @@
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+const BIDDER_CODE = 'colombia';
+const ENDPOINT_URL = 'https://ade.clmbtech.com/cde/prebid.htm';
+//const ENDPOINT_URL = 'https://originqaade.clmbtech.com/cde/prebid.htm';
+const HOST_NAME = document.location.protocol + '//' + window.location.host;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['clmb'],
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.placementId);
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    let payloadArr = []
+    let ctr = 1;
+    validBidRequests = validBidRequests.map(bidRequest => {
+      const params = bidRequest.params;
+      const sizes = utils.parseSizesInput(bidRequest.sizes)[0];
+      const width = sizes.split('x')[0];
+      const height = sizes.split('x')[1];
+      const placementId = params.placementId;
+      const cb = Math.floor(Math.random() * 99999999999);
+      const bidId = bidRequest.bidId;
+      const referrer = (bidderRequest && bidderRequest.refererInfo && bidderRequest.refererInfo.referer) ? bidderRequest.refererInfo.referer : '';
+      let mediaTypes = {}
+      let payload = {
+        v: 'hb1',
+        p: placementId,
+        pos: '~' + ctr,
+        w: width,
+        h: height,
+        cb: cb,
+        r: referrer,
+        uid: bidId,
+        t: 'i',
+        d: HOST_NAME,
+        fpc: params.fpc,
+        _u: window.location.href,
+        mediaTypes: Object.assign({}, mediaTypes, bidRequest.mediaTypes)
+      };
+      if (params.keywords) payload.keywords = params.keywords;
+      if (params.category) payload.cat = params.category;
+      if (params.pageType) payload.pgt = params.pageType;
+      if (params.incognito) payload.ic = params.incognito;
+      if (params.dsmi) payload.smi = params.dsmi;
+      if (params.optout) payload.out = params.optout;
+      if (params.yob) payload.yob = params.yob;
+      if (params.gender) payload.g = params.gender;
+      if (params.floor) payload.fb = params.floor;
+      if (params.currency) payload.cur = params.currency;
+      if (params.bcat) payload.cat = params.bcat;
+      if (params.bcat) payload.cat = params.bcat;
```

Comment:
you are requesting a variety of openrtb params as bidder params, it seems unnecessary.

Reply:
Removed ORTB params, please review.